DigitalLearningSolutions.Data.Tests/DataServices/MultiPageFormDataServiceTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.DataServices
+{
+    using System;
+    using System.Transactions;
+    using Dapper;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Enums;
+    using DigitalLearningSolutions.Data.Models.MultiPageFormData;
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using FluentAssertions;
+    using Microsoft.Data.SqlClient;
+    using NUnit.Framework;
+
+    public class MultiPageFormDataServiceTests
+    {
+        private const string TestJson = "test json";
+
+        private static readonly Guid TestGuid = Guid.NewGuid();
+
+        private readonly MultiPageFormData expectedData = new MultiPageFormData
+        {
+            Id = 1,
+            TempDataGuid = TestGuid,
+            Json = TestJson,
+            Feature = MultiPageFormDataFeature.AddNewCourse.Name,
+            CreatedDate = new DateTime(2022, 06, 14, 14, 20, 12),
+        };
+
+        private SqlConnection connection = null!;
+
+        private IMultiPageFormDataService multiPageFormDataService = null!;
+
+        [SetUp]
+        public void SetUp()
+        {
+            connection = ServiceTestHelper.GetDatabaseConnection();
+            multiPageFormDataService = new MultiPageFormDataService(connection);
+        }
+
+        [Test]
+        public void GetMultiPageFormDataByGuidAndFeature_returns_expected_MultiPageFormData()
+        {
+            // Given
+            using var transaction = new TransactionScope();
+            InsertMultiPageFormData();
+
+            // When
+            var result = multiPageFormDataService.GetMultiPageFormDataByGuidAndFeature(
+                TestGuid,
+                MultiPageFormDataFeature.AddNewCourse.Name
+            );
+
+            // Then
+            result.Should().BeEquivalentTo(expectedData);

```

Comment:
In every test in this file I need to insert a record to the database, either because I'm testing the insert, or because I'm testing querying that record in some way. I don't see how the current setup is worse than UserTestHelper.GetDefaultDelegateUser for example?